CODEOWNERS

```diff
@@ -1 +1 @@
-* @aobolensk @SergeyKonnov
+* @aobolensk @SergeyKonnov @alexmasterdi @megarungle @adudchen @left841
```

Comment:
who is this people?

Reply:
This people are professional Python developers, I think we can trust them.